lib/instance.js

```diff
@@ -14,6 +14,20 @@ module.exports = function instance(input, done) {
   var driverConfig;
   var runner;
 
+  var exitInstance = function (failures, summary) {
+    // only attach this listener if we aren't running in parallel
+    if (!done && process) {
+      process.on('exit', function () {
+        // exit with non-zero status if there were failures
+        process.exit(failures);
+      });
+    }
+    // attempted to run with no matching suites/tests
+    if (!anyTests && done) {
+      done(summary);
```

Comment:
Now accepting `summary` as a parameter because we don't always want the closured `result`. We pass nothing in the case that we should not invoke `mocha.run()`